src/NextGen/DonationForm/Blocks/DonationFormBlock/resources/app/utilities/ConvertFieldAPIRulesToJoi.ts

```diff
@@ -0,0 +1,83 @@
+import Joi, {AnySchema, ObjectSchema} from 'joi';
+import {Field, Form} from '@givewp/forms/types';
+import {__, sprintf} from '@wordpress/i18n';
+import {reduceFields} from './groups';
+
+const requiredMessage = sprintf(
+    /* translators: base error message */
+    __('%s is required.', 'give`'),
+    `{#label}`
+);
+
```

Comment:
I noticed the amount field is not working properly, the label should be `"Donation Amount"` which is hidden by the template.  I will also point out that it's a little strange to refer to a field that doesn't a visible label here but I guess it's within context so not a big deal.

<img width="751" alt="Screen Shot 2022-10-13 at 2 25 55 PM" src="https://user-images.githubusercontent.com/10138447/195677765-13d94db6-9235-485e-b992-d7ff81d6382f.png">

Reply:
I think that the context is fine. I also updated this to say that it's required like the other fields in 47f0b1e.